packages/dm-core/src/utils/filetypes.ts

```diff
@@ -0,0 +1,20 @@
+export const imageFiletypes = [
+  'gif',
+  'jpeg',
+  'png',
+  'apng',
+  'avif',
+  'svg',
+  'webp',
+]
+export const videoFiletypes = [
+  'mp4',
+  'mov',
+  'wmv',
+  'webm',
+  'avi',
+  'flv',
+  'mkv',
+  'mts',
+]
+export const applicationFiletypes = ['pdf']
```

Comment:
Is "application" a bad name? :thinking: 
How about, 'pdf'? Or are there other examples?

Reply:
right, I initially thought I could include things like "doc" and "docx" here as well, but that didn't work. I'll change the name